openfisca_core/formulas.py

```diff
@@ -639,14 +635,15 @@ def calculate_output_add(formula, period):
     return formula.holder.compute_add(period).array
 
 
-def calculate_output_add_divide(formula, period):
-    return formula.holder.compute_add_divide(period).array
-
-
 def calculate_output_divide(formula, period):
     return formula.holder.compute_divide(period).array
 
 
+def calculate_output_first_month(formula, period):
```

Comment:
The `calculate_output` mechanism is a dirty hook, let's try to not invest more in it.
I think we can avoid it by slightly modyfing the tests in france (see the `france` PR)